app/ui/browser-modern/model/active-element.js

```diff
@@ -14,10 +14,12 @@ import Immutable from 'immutable';
 
 import { register } from '../util/record-constructors';
 
+const VERSION = 1;
+
 const ActiveElement = register(Immutable.Record({
```

Comment:
Not sure how. The record naming is part of the record constructor, not the `register` function.